tee/WorkflowBase.py

```diff
@@ -24,7 +24,7 @@ class WorkflowBase(ABC):
     WORK_DIR_ENV = os.getenv("WORK_DIR_ENV", "Prod")
 
     NOT_SCHEDULABLE = ["QUEUED", "INITIALIZING", "RUNNING", "CANCELING"]
-    ALREADY_RAN = ["COMPLETE", "SYSTEM_ERROR", "EXECUTOR_ERROR", "FAILED", "UNKNOWN"]
```

Comment:
we probably still want this so that we don't keep getting SYSTEM_ERROR with every repeated execution of main? We can always use `recall.py` to rerun those jobs?